books/books/spiders/followall.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import re
+from six.moves.urllib.parse import urlparse
+import datetime
+import scrapy
```

Comment:
PEP-8 for imports (first stdlib, blank line, third-part, blank line, project imports)